lib/parseOptions.js

```diff
@@ -74,10 +75,12 @@ export default function parseOptions (params) {
   } else {
     options.errorLogFile = resolve(process.cwd(), options.errorLogFile)
   }
-
-  // Further processing
   options.accessToken = options.managementToken
-  options.content = options.content || JSON.parse(fs.readFileSync(options.contentFile))
+
+  if (!options.content) {
```

Comment:
Should we maybe add a comment here briefly explaining why it was done the way it was done? Just thinking in terms of someone looking at this in two years and wondering why it's a stream.